CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ These changes are available on the `master` branch, but have not yet been releas
 
 - Added `banner` parameter to `ClientUser.edit`.
   ([#2396](https://github.com/Pycord-Development/pycord/pull/2396))
+- Added automatic retry of HTTP Request that fail with a 503 status code.
```

Comment:
Just a little wordy, this might be better:
```md
- HTTP requests that fail with a 503 status are now retried.
```
Maybe also move it to **Changed**

Reply:
I think this is grammatically better?
```diff
- - HTTP requests that fail with a 503 status are now retried.
+ - HTTP requests that fail with a 503 status are now re-tried.
```

Reply:
dictionary says retried